ripd/rip_cli.c

```diff
@@ -1011,7 +1012,11 @@ DEFPY (clear_ip_rip,
 		listnode_add(input, yang_vrf);
 	}
 
-	return nb_cli_rpc("/frr-ripd:clear-rip-route", input, NULL);
+	ret = nb_cli_rpc("/frr-ripd:clear-rip-route", input, NULL);
+
+	list_delete(&input);
```

Comment:
The intended design is that the northbound should deallocate the list of input arguments (to avoid doing that in all `rpc` callbacks). The ConfD and Sysrepo plugins do that, but the CLI client was leaking it on `nb_cli_rpc()`. I should submit another PR for this later (to avoid double free when using ConfD/Sysrepo).